### PR TITLE
Fix Windows go vet error

### PR DESCRIPTION
**Pull request by akalin-keybase**
Run go vet in CI for Windows also.

### CODE REVIEW

File: go/libkb/util_windows.go [hunk 1]
```diff
@@ -81,26 +82,29 @@ func GetDataDir(id GUID, envname string) (string, error) {
...
-	// a Go array of uint16 here, but we don't yet know the length.
-	rawUnicode := (*[1 << 16]uint16)(unsafe.Pointer(pszPath))[:]
-
-	// utf16.Decode crashes without adjusting the slice length
-	pathLen := 0
-	for i, r := range rawUnicode {
-		if r == 0 {
-			pathLen = i
+	var rawUnicode []uint16
+	for i := uintptr(0); ; i++ {
```
Comment (akalin-keybase) [inline go/libkb/util_windows.go]: Rewrote this since I'm not sure how kosher it is to cast to a pointer to array of size larger than the memory block.

Comment (zanderz) [reply go/libkb/util_windows.go]: This is speculative and doesn't prevent the warning, right?

Comment (akalin-keybase) [reply go/libkb/util_windows.go]: It didn't prevent the warning, it was just to remove the possibility of accessing beyond the string end (e.g., by some compiler optimization that reads multiple uint16s at a time)

**Comment by akalin-keybase** [comment]
@zanderz Can you take another look? Figured out how to fix the go vet warning, and also added some unit tests.

**Pull request merged**